.rubocop.yml

```diff
@@ -344,6 +344,9 @@ Style/Proc:
 Style/RaiseArgs:
   Enabled: false
 
+Style/RedundantBegin:
+  Enabled: false
```

Comment:
How come? I'd expect this to be good.

Reply:
Because it does weird stuff to the indentation if you _don't_ use redundant begins and if you _do_, as I did here, this linter complains. If you really think I should do it the other way I can refactor for that tho I guess

Reply:
I think I prefer the weird indentation thing, because it's Ruby-er – is that alright?